browse/templates/abs/abs.html

```diff
@@ -41,7 +41,7 @@
 
   <div class="leftcolumn">
     <div class="subheader">
-      <h1>{% if abs_meta.primary_archive.id != abs_meta.primary_category.id %}{{ abs_meta.primary_archive.name }} > {% endif %}{{ abs_meta.primary_category.name }}</h1>
```

Comment:
Why the change from <h1> to <p> here?

That could have a negative affect on screen readers.